few/lib/epsilon_lexicase.h

```diff
@@ -96,9 +96,8 @@ void epsilon_lexicase(const ExtMat & F, int n, int d,
   for (int i = 0; i<epsilon.size(); ++i)
     epsilon(i) = mad(F.col(i));
 
-vector<int> ind_locs(n);
-if(lex_size){
-    char message[200];
+  vector<int> ind_locs(n);
```

Comment:
ind_locs should start out empty if you are pushing back values into it. otherwise it will be 2n after lex_size.